### ISSUE TITLE
Images are cropped

### ISSUE THREAD

**Issue by Vincent-Maladiere** [probabl-ai/skore#888]
### Describe the bug

Images get cropped on the left-side of the UI, not sure why

On my notebook:
<img width="610" alt="Screenshot 2024-12-09 at 15 57 41" src="https://github.com/user-attachments/assets/c7dab42c-873e-49b7-a63d-b555d9de0afd">

On skore UI:
<img width="730" alt="Screenshot 2024-12-09 at 15 57 49" src="https://github.com/user-attachments/assets/bb2b9954-46c5-41c8-979c-0aed021ac990">

### Steps/Code to Reproduce

Simply put this image into a project to reproduce:
<img width="610" alt="Screenshot 2024-12-09 at 15 57 41" src="https://github.com/user-attachments/assets/e2edfadc-b704-4de9-8e21-049564471166">

### Expected Behavior

No cropping

### Actual Behavior

Cropping

### Environment

```shell
skore: 0.4.1
```

**Comment by rouk1** [comment]
This seems to be related with seaborn usage.
